src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
```

Comment:
It's better to use the Object.keys() or Object.values() methods to iterate over 'robot' keys (and values) to avoid potential issues with the prototype chain